types/kdbush/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// Type definitions for kdbush 1.0
+// Project: https://github.com/mourner/kdbush
+// Definitions by: DenisCarriere <https://github.com/DenisCarriere>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+type Points = number[][];
```

Comment:
Also including a `z` value in your points is also valid and does not require having the `getX` & `getY` callbacks.

```javascript
// z coordinate (does not require Get callbacks)
const pointsZ = [[110, 60, 4000], [130, 40, 3000]];
kdbush(pointsZ);
```